packages/core/src/common/keys.ts

```diff
@@ -53,6 +53,48 @@ export class KeyCode {
         }
     }
 
+    /**
+     * Parses a string and returns a KeyCode object.
+     * @param keybinding String representation of a keybinding
+     */
+    public static parse(keybinding: string): KeyCode | undefined {
+        const sequence: string[] = [];
+
+        const keys = keybinding.split('+');
+        /* If duplicates i.e ctrl+ctrl+a or alt+alt+b or b+alt+b it is invalid */
+        if (keys.length !== new Set(keys).size) {
+            return undefined;
+        }
+
+        for (let keyString of keys) {
+            if (SPECIAL_ALIASES[keyString] !== undefined) {
+                keyString = SPECIAL_ALIASES[keyString];
+            }
+            const key = EASY_TO_KEY[keyString];
+
+            if (keyString === 'meta') {
```

Comment:
Could meta just be an easy key too ?

Reply:
Actually for meta/command it's a bit of a workaround. I can't map it to easy key as the keyCode [is browser dependent](https://stackoverflow.com/a/3922353)... So instead theia already has the logic on keydown to check if the meta key was pressed in `createKeyCode` with the help of the `Modifier` map in `keys.ts`.

I think it's easier to just handle it as a separate case here. I could add `cmd` to the special aliases though. 

With the way it works right now, that means a mac user could bind commands using `cmd`, `command` or `meta` and they would work on mac os, but if he switches to Windows/Linux, they would still work (with ctrl instead of command). I'm not sure I answered correctly, so let me know if that's what you meant.

Reply:
OK I think it just caught my attention.. from my c/cpp that const key = EASY_TO_KEY[keyString]; is called with an invalid keyString.

But I guess it's OK.